src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -413,11 +412,11 @@ def __modify_updates(self, job_id_list, update_type):
             raise ValueError("Unknown request")
 
         self._jm.modify_job_refresh(job_id_list, update_adjust)
-        output_states = self._jm.get_job_states(job_id_list)
-
         if update_adjust == 1:
             self.start_job_status_loop()
-            self.send_comm_message(STATUS, output_states)
+
+        output_states = self._jm.get_job_states(job_id_list)
+        self.send_comm_message(STATUS, output_states)
```

Comment:
altered this slightly to send out job status messages regardless of whether the adjustment was start or stop